test/definitions/logging/suse155-logs.json

```diff
@@ -0,0 +1,34 @@
+{
+    "global_tags": {
+        "owning_team": "virtuoso",
+        "Environment": "development",
+        "Department": "product",
+        "Product": "virtuoso"
+    },
+    "resources": [
+        {
+            "id": "suse5-logs",
+            "provider": "aws",
+            "type": "ec2",
+            "size": "t3.nano",
+            "ami_name": "suse-sles-15-sp5-chost-byos-v????????-hvm-ssd-x86_64"
+        }
+    ],
+    "instrumentations": {
+        "resources": [
+            {
+                "id": "nr_infra_suse155",
+                "resource_ids": [
+                    "suse5-logs"
+                ],
+                "provider": "newrelic",
+                "source_repository": "https://github.com/newrelic/open-install-library",
+                "deploy_script_path": "test/deploy/linux/newrelic-cli/install/roles",
+                "params": {
+                    "validate_output": "Logs Integration\\s+\\(installed\\)",
```

Comment:
It looks you're not specifying the logs integration target to install, could you please check ? 

For reference, please have a look at this - https://github.com/newrelic/open-install-library/blob/main/test/definitions/logging/amazonlinux2-logs.json#L26